pipelines/build/common/config_regeneration.groovy

```diff
@@ -163,12 +163,11 @@ class Regeneration implements Serializable {
     * @param configuration
     */
     List<String> getTestList(Map<String, ?> configuration) {
-        if (configuration.containsKey("test")) {
+        if (configuration.containsKey("test") && configuration.get("test")) {
             if (isMap(configuration.test)) {
                 return (configuration.test as Map).get("nightly") as List<String> // no need to check for release
-            } else {
-                return configuration.test as List<String>
             }
+            return ['sanity.openjdk', 'sanity.system', 'extended.system', 'sanity.perf', 'sanity.external']
```

Comment:
this looks like a constant, should it be extracted and named?

Reply:
updated.